### ISSUE TITLE
[Feature request] Constrain PostFX Volume to specific camera

### ISSUE THREAD

**Issue by NoCultist** [FlaxEngine/FlaxEngine#2573]
**Issue description:**
When using PostFX Volume in some cases it would be useful to constrain postprocessing to certain camera.
example: in fps game render scene with depth of field but player weapon without it but still affected by different PostFX Volume.
It could be done via directly connecting specific camera to the volume or by volume affecting only cameras on the same layer as it itself is.

**Comment by mafiesto4** [comment]
This can be achieved by attaching PostFx volume to the specific camera and making it small enough to contain only that camera.

**Comment by xxSeys1** [comment]
What if you want to use 2 cameras to render the same thing (aka both being in the same place) just one camera renders with f.e. dof and one without then u composite.

**Comment by mafiesto4** [comment]
Good idea, we could use the existing Layers system (Camera's `RenderLayersMask`).

**Comment by mafiesto4** [comment]
Added in 7018666a8cb10a3dfe3a29e86a4298bf8b498fc9.
